test/test_torch.py

```diff
@@ -17938,22 +17938,14 @@ def test_strided_mm_bmm(self, device, dtype):
 
     @skipCUDAIf(torch.version.cuda == "10.1", "flaky on CUDA 10.1")
     @onlyOnCPUAndCUDA
-    @dtypes(*torch.testing.get_all_fp_dtypes(), *torch.testing.get_all_complex_dtypes())
+    @dtypes(*torch.testing.get_all_fp_dtypes(include_bfloat16=AMPERE_OR_ROCM), *torch.testing.get_all_complex_dtypes())
```

Comment:
I think the original dtypes is
```python
@dtypes(*torch.testing.get_all_fp_dtypes(), *torch.testing.get_all_complex_dtypes())
```

Reply:
right but bfloat16 is only supported for ampere and rocm